compressor/filters/css_default.py

```diff
@@ -104,3 +104,28 @@ def url_converter(self, matchobj):
 
     def src_converter(self, matchobj):
         return self._converter(matchobj, 2, "src=%s%s%s")
+
+
+class CssRelativeFilter(CssAbsoluteFilter):
+    """
+    Do similar to ``CssAbsoluteFilter`` URL processing
+    but replace ``settings.COMPRESS_URL`` prefix with '../../'.
+    """
+
+    def add_suffix(self, url):
+        url = super(CssRelativeFilter, self).add_suffix(url)
+        old_prefix = self.url
+        if self.has_scheme:
+            old_prefix = '{}{}'.format(self.protocol, old_prefix)
+        # One level up from 'css' / 'js' folder
+        new_prefix = ['..']
+        # N levels up from settings.COMPRESS_OUTPUT_DIR:
+        # - if it's 'CACHE' (default) then it'd be ['..'];
+        # - if it's 'CACHE/in/depth' then it'd be ['..', '..', '..'].
+        new_prefix += [
+            '..' for part in os.path.normpath(
+                settings.COMPRESS_OUTPUT_DIR
+            ).split(os.sep) if part
+        ]
+        new_prefix = '/'.join(new_prefix)
```

Comment:
In general - looks better, thanks! I'll use your code ;)

> the `if part` shouldn't be necessary when using `normpath`

Due to `django-compressor` implementation, the following cases will be treated the same:

* `settings.COMPRESS_OUTPUT_DIR = 'CACHE'
* `settings.COMPRESS_OUTPUT_DIR = '/CACHE'

So `normpath` doesn't solve the problem. Unless I'll do some conditional replacement.

`(os.sep + '..')` is wrong; `new_prefix` should be a URL, not file path.